src/vs/workbench/browser/parts/editor/editorParts.ts

```diff
@@ -231,6 +231,14 @@ export class EditorParts extends Disposable implements IEditorGroupsService, IEd
 		return this.activePart.arrangeGroups(arrangement);
 	}
 
+	toggleGroupArrangement(): void {
+		return this.activePart.toggleGroupArrangement();
```

Comment:
Once we have a optional `group: IEditorGroupView` passed in, we can also update this method to `this.getPart(group)`

Reply:
Toggle would probably work fine by just applying it to the activePart.
However, it seems to me like arrangeGroups might struggle from it's current behaviour. You are also not passing on the target which I think is necessary.